src/scripts/main.js

```diff
@@ -18,10 +19,62 @@ const food = {
   },
 };
 
-const tree = document.querySelector('#tree');
+// const tree = document.querySelector('#tree');
+
+// function createTree(element, data) {
+//   if (!Object.keys(data).length) {
+//     return;
+//   }
+
+//   const ul = document.createElement('ul');
+
+//   for (const key in data) {
+//     if (data.hasOwnProperty(key)) {
+//       const li = document.createElement('li');
+
+//       li.textContent = data;
+
+//       createTree(li, data[key]);
+
+//       ul.appendChild(li);
+//     }
+//   }
+
+//   element.appendChild(ul);
+// }
+
+// createTree(tree, food);
```

Comment:
This section of code is commented out but seems to be a duplicate of the active `createTree` function. Consider removing it if it's not needed to avoid confusion.